### ISSUE TITLE
GoogleAnalytics.getInstance() is blocking

### ISSUE THREAD

**Issue by GoogleCodeExporter** [gzog/analytics-issues#552]
```

Name of affected component: Mobile Tracking

Name of related library and version, if applicable (e.g. Android, iOS,
Snippets, etc.): Android - Google Play services - 6.5.87

Issue summary:
GoogleAnalytics.getInstance() is blocking which freezes the app

Steps to reproduce issue:
1. GoogleAnalytics.getInstance() in main thread, its blocked, and app freezes


Notes:
Many devs facing the same issue, the workaround given in SO does fix the 
issues, i.e remove the config.xml from AndroidManifest and set it using JAVA 
code.

Ref : 
http://stackoverflow.com/questions/27423218/android-googleanalytics-getinstance

```

Original issue reported on code.google.com by `mu...@authenticatorplus.com` on 18 Dec 2014 at 6:10

**Comment by GoogleCodeExporter** [comment]
```
For ref, below the stack when app is blocked  

at com.google.android.gms.analytics.ae.getLogger(Unknown Source)  
  at com.google.android.gms.analytics.ae.W(Unknown Source)  
  at com.google.android.gms.analytics.z$a.f(Unknown Source) 
  at com.google.android.gms.analytics.n.a(Unknown Source)   
  at com.google.android.gms.analytics.n.x(Unknown Source)   
  at com.google.android.gms.analytics.GoogleAnalytics.eZ(Unknown Source)    
  at com.google.android.gms.analytics.GoogleAnalytics.<init>(Unknown Source)    
  at com.google.android.gms.analytics.GoogleAnalytics.<init>(Unknown Source)    
  at com.google.android.gms.analytics.GoogleAnalytics.getInstance(Unknown Source)   
  at com.mufri.authenticatorplus.AuthenticatorApplication.getTracker(AuthenticatorApplication.java:228) 
  at com.mufri.authenticatorplus.testability.BaseActivity.onCreate(BaseActivity.java:30)    
  at com.mufri.authenticatorplus.StartupActivity.onCreate(StartupActivity.java:80)  

```

Original comment by `mu...@authenticatorplus.com` on 18 Dec 2014 at 6:10

**Comment by GoogleCodeExporter** [comment]
```
Please see https://code.google.com/p/android/issues/detail?id=82157, 
https://productforums.google.com/forum/#!category-topic/analytics/report-bugs/RA
gDNYFk6jo  and this answer on stackoverflow: 
http://stackoverflow.com/questions/27533679/google-analytics-blocks-android-app/
27542483#27542483.
```

Original comment by `jaredsburrows@gmail.com` on 11 Feb 2015 at 12:59

**Comment by GoogleCodeExporter** [comment]
```
Issue 565 has been merged into this issue.
```

Original comment by `pfrise...@google.com` on 21 Mar 2015 at 12:06

**Comment by GoogleCodeExporter** [comment]
```
Fixed in Release Version 4.0.6 - Google Play Services 7.0 (March 19, 2015)
see 
https://developers.google.com/analytics/devguides/collection/android/changelog
```

Original comment by `pfrise...@google.com` on 23 Mar 2015 at 4:06
- Changed state: **Fixed**
